pkgs/development/libraries/libnotify/default.nix

```diff
@@ -22,10 +22,20 @@ stdenv.mkDerivation rec {
   patches = [
     # Fix darwin build
     # https://gitlab.gnome.org/GNOME/libnotify/merge_requests/9
-    (fetchpatch {
-      url = "https://gitlab.gnome.org/GNOME/libnotify/commit/55eb69247fe2b479ea43311503042fc03bf4e67d.patch";
-      sha256 = "1hlb5b7c5axiyir1i5j2pi94bm2gyr1ybkp6yaqy7yk6iiqlvv50";
-    })
+    (
+      fetchpatch {
+        url = "https://gitlab.gnome.org/GNOME/libnotify/commit/55eb69247fe2b479ea43311503042fc03bf4e67d.patch";
+        sha256 = "1hlb5b7c5axiyir1i5j2pi94bm2gyr1ybkp6yaqy7yk6iiqlvv50";
+      }
+    )
+
+    # add support for replacing an existing notification
+    (
+      fetchpatch {
+        url = "https://launchpadlibrarian.net/105791133/print-and-replace-id-v3.patch";
+        sha256 = "1lfk5msv81brz7f7srxi4s4l02nasskzmpq0b9ch8hbn79z86g0b";
+      }
+    )
   ];
```

Comment:
can we format this like how it was before? This looks like what nixpkgs-fmt tries to do which is a bug.

Reply:
Actually, upstream is open to accepting the change there, so I'd rather handle this properly where it belongs.